packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -788,6 +792,12 @@ abstract class TestRunner {
     await this.remoteStore.enableNetwork();
   }
 
+  private async doAcquirePrimaryLease(): Promise<void> {
+    expect(this.queue.containsDelayedOperation(TimerId.ClientStateRefresh)).to
+      .be.true;
```

Comment:
Doesn't matter since this code should go away, but FYI- runDelayedOperationsEarly() will throw if the timer doesn't exist, so this check should be unnecessary.